bookmark.h

```diff
@@ -0,0 +1,56 @@
+#ifndef BOOKMARK_H
+#define BOOKMARK_H
+#include <string>
+#include <iostream>
+#include <conio.h>
+
+using namespace std;
+class Bookmark {
+        public:
+            void displayMainMenu();
+            void addUser(int userCounter);
+            void editUser(int userCounter);
+            void viewAllUsers(int userCounter);
+            void deleteUser(int userCounter);
+
+            void addBook(int bookCounter);
+            void deleteBook(int bookCounter);
+            void editBook(int bookCounter);
+            void searchBook(int bookCounter);
+            void viewAllBooks(int bookCounter);
+            void displayTopBooks();
+
+
+//counter for book array and user array
```

Comment:
no need for this comment. The code is self-explanatory enough. Maybe this helps -> https://app.works/good-comments-vs-bad-comments/#:~:text=Conclusion-,Why%20are%20most%20comments%20bad%3F,comments%20need%20to%20be%20there.